specification/cognitiveservices/data-plane/LUIS/Authoring/preview/v3.0/LUIS-Authoring.json

```diff
@@ -8591,6 +8893,125 @@
         }
       }
     },
+    "LuisAppV2": {
+      "description": "Exported Model - An exported LUIS Application.",
+      "additionalProperties": true,
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the application.",
+          "type": "string"
+        },
+        "versionId": {
+          "description": "The version ID of the application that was exported.",
+          "type": "string"
+        },
+        "desc": {
+          "description": "The description of the application.",
+          "type": "string"
+        },
+        "culture": {
+          "description": "The culture of the application. E.g.: en-us.",
+          "type": "string"
+        },
+        "intents": {
+          "description": "List of intents.",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/HierarchicalModelV2"
+          },
+          "type": "array"
+        },
+        "entities": {
+          "description": "List of entities.",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/HierarchicalModelV2"
+          },
+          "type": "array"
+        },
+        "closedLists": {
+          "description": "List of list entities.",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/ClosedList"
+          },
+          "type": "array"
+        },
+        "composites": {
+          "description": "List of composite entities.",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/HierarchicalModelV2"
+          },
+          "type": "array"
+        },
+        "hierarchicals": {
+          "description": "List of hierarchical entities.",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/HierarchicalModelV2"
+          },
+          "type": "array"
+        },
+        "patternAnyEntities": {
+          "description": "List of Pattern.Any entities.",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/PatternAny"
+          },
+          "type": "array"
+        },
+        "regex_entities": {
```

Comment:
It's unusal to see snake case on a JSON payload, especially since the rest is in CamelCase (like patternAnyEntities). Are you sure it's the correct name?

Reply:
I reviewed the naming again and they are correct